assets/xml/scenes/dungeons/ganon_final.xml

```diff
@@ -1,5 +1,6 @@
 <Root>
     <File Name="ganon_final_scene" Segment="2">
+        <Path Name="ganon_final_scenePathway_0001F4" Offset="0x01F4" NumPaths="6"/>
```

Comment:
I think there are only 4 here?

Reply:
yes, my math was off. will correct when renaming to PathList for review